test/e2e/storage/persistent_volumes.go

```diff
@@ -281,6 +281,7 @@ var _ = utils.SIGDescribe("PersistentVolumes", func() {
 				framework.ExpectNoError(framework.WaitForPodSuccessInNamespace(c, pod.Name, ns))
 
 				By("Deleting the claim")
+				framework.ExpectNoError(framework.DeletePodWithWait(f, c, pod))
```

Comment:
Pod should also be deleted after L299.  I don't understand why this isn't causing the test to fail
/lgtm cancel

Reply:
Oh I see. The test PVC, PV cleanup method doesn't actually wait for the object to be gone.

Reply:
So do we still need the delete at the end, since the test doesn't care what happens after Pod success? Or will this prevent PV/PVC cleanup after It()?

Reply:
I think we should still add it.  I think it would be good (in a separate PR) to improve the cleanup routines